csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1019,15 +1024,15 @@ In addition, the `document` object may provide the 7 optional properties Acknowl
 
 #### 3.2.1.1 Document Property - Acknowledgments
 
-List of acknowledgments (`acknowledgments`) of value type `array` with 1 or more items of type Acknowledgment contains a list of acknowledgment elements.
+List of acknowledgments (`acknowledgments_t`) of value type `array` with 1 or more items of type Acknowledgment contains a list of acknowledgment elements.
 
     "acknowledgments": {
       // ...
     },
 
 #### 3.2.1.2 Document Property - Aggregate Severity
 
-Aggregate severity (`aggregate_severity`) of value type `object` with the mandatory property `text` and the optional property `namespace` is a vehicle that is provided by the document producer to convey the urgency and criticality with which the one or more vulnerabilities reported should be addressed. It is a document-level metric and applied to the document as a whole — not any specific vulnerability. The range of values in this field is defined according to the document producer's policies and procedures.
+Aggregate severity (`aggregate_severity`) of value type `object` with the mandatory property `text` and the optional property `namespace` is a vehicle that is provided by the document producer to convey the urgency and criticality with which the reported issues should be addressed. It is a document-level metric and applied to the document as a whole — not any specific vulnerability. The range of values in this field is defined according to the document producer's policies and procedures.
```

Comment:
Schema updated.

Reply:
@mprpic, @sthagen, @santosomar: Please provide feedback.

Reply:
Three times then :wink:. @tlimmer Please "keep" `one or more vulnerabilities` because then we can simply merge, and I can continue to work on the document/schema - thanks.

Reply:
Or maybe this way: When tomorrow afternoon European time this PR is still using the "issues" term in here, I will simply merge to be able to continue collecting all other comments into the main revision. Thanks.

Reply:
Is reverted with #206.